### PR TITLE
Fix sample code in the doc

### PR DESCRIPTION
**Pull request by zliang-akamai**

### CODE REVIEW

**Review by bflad — APPROVED** [review]
Looks good to me, thank you for this fix, @zliang-akamai 🚀

**Comment by bflad** [comment]
@zliang-akamai please note we unfortunately cannot merge this until the CLA is signed as noted above by the bot.

**Comment by zliang-akamai** [comment]
> @zliang-akamai please note we unfortunately cannot merge this until the CLA is signed as noted above by the bot.

Thank you! Working getting approval from my employer.

**Pull request merged**
